src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double CAPACITY_MULTIPLIER = 1.5;
+    private int capacity;
+    private int size;
+    private T[] values;
+
+    public ArrayList() {
```

Comment:
Don't begin class or method implementation with an empty line. The empty line at the start of the constructor is unnecessary and should be removed.